templates/web/pages/product_edit/product_edit_form_display.tt.html

```diff
@@ -390,4 +390,5 @@ <h1>[% title %]</h1>
     opacity: 0.7;
 }
 </style>
+<script type="text/javascript" src="/js/product_edit_form.js"></script>
```

Comment:
Hi @MonalikaPatnaik , there's a name mismatch for the script, the name of the file checked in is product_edit_form_display.js and not product_edit_form.js

Also gulpfile.ts generates some "dist" version of the JS scripts in the /js/dist/ directory, could you add your new scripts to gulpfile.ts and then load them from /js/dist ?

gulpfile.ts currently contains this:

```
const jsSrc = [
  "./html/js/display*.js",
  "./html/js/product-multilingual.js",
  "./html/js/search.js",
  "./html/js/hc-sticky.js",
  "./html/js/stikelem.js",
  "./html/js/scrollNav.js",
];

```